src/components/Organizations/Notifications/index.js

```diff
@@ -185,14 +214,23 @@ const OrgNotifications = ({
               </div>
             </NameTagCol>
             <div className="notifdata">
-              Webhook: {project.webhook}
-              <br></br>
-              Channel: {project.channel}
+              {renderWebbook(project.webhook, () => {
+                setEditState({ open: false, current: project });
+              })}
+              <p> Channel: {project.channel}</p>
             </div>
             <div className="actions">
```

Comment:
Noticed the `svgs` for these action buttons aren't sized consistently with the other tabs. The others are pulling the sizing from [TableActions](https://github.com/uselagoon/lagoon-ui/blob/main/src/components/Organizations/SharedStyles.tsx#L278), which isn't being used on the `notifications` tab.

Notifications:
![image](https://github.com/uselagoon/lagoon-ui/assets/40746380/4548b84f-0e0b-45b8-8dbe-d872368f64bb)

Groups:
![image](https://github.com/uselagoon/lagoon-ui/assets/40746380/ff61c48c-3c7b-42ca-aa03-d9a393375684)